### ISSUE TITLE
Calendar sharing 

### ISSUE THREAD

**Issue by BobWs** [bjuretko/docker-baikal-infcloud#25]
Is there a way to share Calendars?

Maybe you can check this out http://sabre.io/dav/caldav-sharing/

**Comment by alfureu** [comment]
Not sure about your user case, but there is also this: https://github.com/pimutils/vdirsyncer

**Comment by BobWs** [comment]
Thanks but that is not what I was looking for, but I found a solution...

https://github.com/sabre-io/Baikal/issues/743

I followed the instructions and added this to the server.php file
`$this->server->addPlugin(new \Sabre\DAV\Sharing\Plugin());`
`$this->server->addPlugin(new \Sabre\CalDAV\SharingPlugin());`

And I am now able to share Calendars between users.

Original post how to edit the file, but as of baikal 0.5.2 the database part isn’t necessary to alter. 
https://github.com/sabre-io/Baikal/issues/529#issuecomment-232803267

I have done this with the regular version of baikal (not the docker version)

**Comment by alfureu** [comment]
where exactly did you find the `server.php` file?

**Comment by de-es** [comment]
@DOFfactory see https://github.com/sabre-io/Baikal/pull/801

**Comment by BobWs** [comment]
> where exactly did you find the `server.php` file?

The file can be found in /baikal/Core/Frameworks/Baikal/Core/Server.php
This is in the regular version of baikal not the docker version.

**Comment by arsphl** [comment]
I just tested the version 0.6.1 of Baikal locally on my laptop. Sharing seems to work fine once you have understood that it creates a new resource under a UUID name in the calendars of the account you have shared it with. Being on a laptop I did not set-up email forwarding with sendmail or any php sendmail options.

Can someone explain a bit more what can be done with the Share with (uri): field of the action section? 

I suppose that entering a mailto:<email address> is to send an email. But there is probably more to it.
Thanks
